x-pack/plugins/cross_cluster_replication/public/app/components/form_entry_row.js

```diff
@@ -50,6 +50,7 @@ export class FormEntryRow extends PureComponent {
     ]),
     disabled: PropTypes.bool,
     areErrorsVisible: PropTypes.bool.isRequired,
+    dataTestSubj: PropTypes.string,
```

Comment:
What do you think of using the spread operator to support pass-through props? This is something all EUI components do. So this line would become `...rest` and then...

Reply:
a gripe I've always had with that method is the need to list out every expected property before being able to use `...rest` for `data-test-subject`, and there's no clarity on _which_ child component will actually receive the `...rest` properties

 for EUI, it makes sense since it's a component library, but our app components are pretty specialized so I think using `...rest` is overkill. however, I am going to rename this property to just `testSubj` 😄